packages/docusaurus-plugin-content-docs/src/index.ts

```diff
@@ -198,10 +198,13 @@ export default function pluginContentDocs(
             previousId,
             nextId,
           } = sidebarsUtils.getDocNavigation(doc.id);
-          const toDocNavLink = (navDocId: string): DocNavLink => ({
-            title: docsBaseById[navDocId].title,
-            permalink: docsBaseById[navDocId].permalink,
-          });
+          const toDocNavLink = (navDocId: string): DocNavLink => {
+            const navDoc = docsBaseById[navDocId];
```

Comment:
Maybe apply destructuring assignment here?

Reply:
Added a bit of destructuring but I'd rather not destructure frontmatter, as it's easier to see in the final "algo" that we pick the sidebar label from frontmatter (also annoying to have to "rename" the `sidebar_label` to `sidebarLabel` to make eslint happy)